### ISSUE TITLE
Some ternary expressions cause a crash

### ISSUE THREAD

**Issue by jrossignol** [jrossignol/ContractConfigurator#281]
```
targetBody = @/targetPlanet

   DATA
   {
       type = CelestialBody
       targetPlanet = HomeWorld().Parent() == Sun ? HomeWorld().Children().Random() : HomeWorld().Parent().Children().Random()
   }
```

**Comment by jrossignol** [comment]
:fu: to Mono, this has to be a bug in the interpreter.  Spent way too much time getting nowhere with this one.

**Comment by jrossignol** [comment]
Fixed by fae1266
